README.md

```diff
@@ -14,13 +14,32 @@ TODO
 
 \subsubsection*{Cantor set}
 \subsubsection*{Cartesian space ${\mathbb R}^n$}
+The set of all $n$-tuples, $\{ (x_1, x_2, ... , x_n) : x_i \in \mathbb{R} \}$.\\
+The number $n$ is called the dimension of $\mathbb{R}^n$, and the $n$-tuples
+can be thought of as vectors, since they form a natural vector space.\\
+The set is typically augmented with the "Usual Topology", or sometimes the
+"Euclidean Topology", whose basis is the set of all balls,
+$\{ (x_1, x_2, ..., x_n) : \sqrt{x_1^2 + x_2^2 + ... + x_n^2} < r, r > 0 \}$.
+Equivalently, the topology can be defined so that a subset $U$ is open if and only
+if for all $u\in U$, there is a ball $B_r$ so that $u \in B_r \subseteq U$.
+If no topology on the space is mentioned it is assumed to be the Usual.
+
 \subsubsection*{Cell complex}
 \subsubsection*{Cell $e^n$}
 \subsubsection*{Cell structure}
 \subsubsection*{Component}
 \subsubsection*{Compact}
-\subsubsection*{Cone}
-\subsubsection*{Contractible space}
+\subsubsection*{Cone} The Cone of a space $X$, is thought of as extending the
+space into a cylinder, and then collapsing one end to a point.  So the Cone
+is $CX = (X \times I) / (X \times \{0\})$.  
+
+
+
+\subsubsection*{Contractible space} A Topological Space is called Contractible
+iff the identity map on the space is homotopic to a constant map.  For example,
+the Euclidean space of any dimension is Contractible to any point,
+the circle is not Contractible, a disconnected set is not Contractible.  
```

Comment:
How come this definition doesn't involve the null-homotopy?

Reply:
Same thing, a space is contractible if the identity is null-homotopic, i just forgot to mention it.